modules/chassis-aws-analytics/manifests/init.pp

```diff
@@ -2,42 +2,28 @@
 	$config
 ) {
 
-	# Install Docker
-	include 'docker'
-
-	# Install Docker Compose
-	class { 'docker::compose':
-		ensure => present,
-	}
-
-	if ( ! empty( $config[disabled_extensions] ) and 'humanmade/chassis-aws-analytics' in $config[disabled_extensions] ) {
-		$dir = absent
-		$compose_file = absent
+	# Allow disabling the extension
+	if ( !empty($config[disabled_extensions]) and 'humanmade/chassis-aws-analytics' in $config[disabled_extensions] ) {
+		$status = stopped
+		$active = false
 	} else {
-		$dir = directory
-		$compose_file = present
+		$status = running
+		$active = true
 	}
 
-	# Add the docker-compose.yaml file.
-	file { '/home/vagrant/aws-analytics':
-		ensure => $dir,
-	}
-	file { '/home/vagrant/aws-analytics/docker-compose.yaml':
-		ensure  => $compose_file,
-		mode    => '0777',
-		content => template('chassis-aws-analytics/docker-compose.yaml'),
-		require => [
-			File['/home/vagrant/aws-analytics'],
-		],
+	# Endpoints store.
+	file { '/tmp/endpoints':
```

Comment:
This may get wiped on reboot, as I think `/tmp` is a tmpfs; can it be handled in whatever's using it instead?

Reply:
Hmm yeah I can move the directory creation into server.js. It was intentional originally but may as well persist the files